.github/workflows/_deploy.yml

```
@@ -60,6 +60,9 @@ jobs:
           VITE_REACT_APP_MERMAID_COLLECT_LINK: ${{ inputs.vite_react_app_mermaid_collect_link }}
           VITE_REACT_APP_MERMAID_API_ENDPOINT: ${{ inputs.vite_react_app_mermaid_api_endcpoint }}
           VITE_REACT_APP_MAP_TILER_API_KEY: ${{ inputs.vite_react_app_map_tiler_api_key }}
+          VITE_ENVIRONMENT: ${{ inputs.environment }}
+          VITE_PUBLIC_POSTHOG_KEY: ${{ inputs.vite_public_posthog_key }}
```

actionlint (1.7.7)

property "vite_public_posthog_key" is not defined in object type {domain: string; environment: string; vite_react_app_auth0_audience: string; vite_react_app_auth0_client_id: string; vite_react_app_auth0_domain: string; vite_react_app_map_tiler_api_key: string; vite_react_app_mermaid_api_endcpoint: string; vite_react_app_mermaid_collect_link: string}

(expression)